nss-day-cohort-50/rare-rambunctious-rattlers-server
# mc-create-tag

## Functionality
In Postman, we can now POST new tags

### *How to test*
Server-side only, add a tag in Postman using the POST methods and the url ending in the resource "tags".
To verify that it worked, use a GET method on same url.
